README.md

```diff
@@ -1,42 +1,75 @@
 [![Build Status](https://travis-ci.org/JSJitsu/ai-battle-website.svg?branch=master)](https://travis-ci.org/JSJitsu/ai-battle-website)
-# Javascript Battle
+# Javascript Fight Club
 
-Every day there is an epic battle between legendary heroes.  Control your hero with either a pre-defined function from our repo or your own submitted function (your hero’s “brain”), and see the results of each day’s battle as your hero’s prowess is tested against the competition.
+Tool to run the games that the website is able to replay. Real or test data can be used to generate a battle.
```

Comment:
I'd make this more of a complete sentence. We don't have to make it the story version the first one was, but I'd suggest more like...

```
This repository contains the code that powers the website. 
By forking this, you will be able to develop the site code, or 
use real (github player ai code) or test (just local versions of 
those scripts) data to generate a battle.